src/components/templates/Audience/Audience.tsx

```diff
@@ -300,11 +300,11 @@ export const Audience: React.FunctionComponent = () => {
   // @debt this return useMemo antipattern should be rewritten
   return useMemo(() => {
     const takeSeat = (row: number | null, column: number | null) => {
-      if (!venueId || !userUid) return;
+      if (!venueId || !userId) return;
 
       makeUpdateUserGridLocation({
         venueId,
-        userUid,
+        userUid: userId,
```

Comment:
[OOS] Do we agree on using userId everywhere with deprecating userUid?

Reply:
Personally I think we should. We only ever used userUid to align with what firebase’s underlying auth calls it, but that’s not a good reason to use it IMO; and our far more common patter n has been userId

Reply:
Added `@debt` comments to explain the commented code blocks in https://github.com/sparkletown/sparkle/pull/1323/commits/5f2acbd92b4f96c175bf39ab1ffc0c0bb7d92624

Reply:
Sounds good 👍